database/insertData/03_template_F_companyJoin.js

```diff
@@ -0,0 +1,319 @@
+                  ]
+                }
+                parameterQueries: {
+                  user_id: {
+                    operator: "objectProperties"
+                    children: ["applicationData.userId"]
+                  }
+                  organisation_id: {
+                    operator: "objectProperties"
+                    children: ["applicationData.responses.S1Q1.selection.id"]
+                  }
+                }
+              }
+              {
+                actionCode: "grantPermissions"
+                trigger: ON_REVIEW_SUBMIT
+                sequence: 130
+                condition: {
+                  operator: "AND"
+                  children: [
+                    {
+                      operator: "="
+                      children: [
+                        {
+                          operator: "objectProperties"
+                          children: [
+                            "applicationData.reviewData.latestDecision.decision"
+                          ]
+                        }
+                        "CONFORM"
+                      ]
+                    }
+                    {
+                      operator: "objectProperties"
+                      children: ["applicationData.reviewData.isLastLevel"]
+                    }
+                  ]
+                }
+                parameterQueries: {
+                  username: {
+                    operator: "objectProperties"
+                    children: ["applicationData.username"]
+                  }
+                  orgName: {
+                    operator: "objectProperties"
+                    children: ["applicationData.responses.S1Q1.selection.name"]
+                  }
+                  permissionNames: ["canApplyDrugRego"]
+                }
+              }
+              {
+                actionCode: "changeStatus"
+                trigger: ON_REVIEW_SUBMIT
+                sequence: 110
+                condition: {
+                  operator: "AND"
+                  children: [
+                    {
+                      operator: "="
+                      children: [
+                        {
+                          operator: "objectProperties"
+                          children: [
+                            "applicationData.reviewData.latestDecision.decision"
+                          ]
+                        }
+                        "CONFORM"
+                      ]
+                    }
+                    {
+                      operator: "objectProperties"
+                      children: ["applicationData.reviewData.isLastLevel"]
+                    }
+                  ]
+                }
+                parameterQueries: {
+                  applicationId: {
+                    operator: "objectProperties"
+                    children: ["applicationData.applicationId"]
+                  }
+                  newStatus: "Completed"
+                }
+              }
```

Comment:
Out of order - can move to sequential place please?

Reply:
Sure, done c28a1c66